### PR TITLE
Fixes the profile.json URL contained in our `authResponse`

### PR DESCRIPTION
**Pull request by hstove**
Fix for #1763

### CODE REVIEW

**Comment by markmhx** [comment]
@hstove we need to assign this over to someone for code review?

**Comment by hstove** [comment]
Correct, I've added Ken and Matt for review.

**Comment by zone117x** [comment]
Should we add a unit test for this situation to prevent future regressions, or is this too trivial of an issue for that?

File: app/js/auth/index.js [hunk 1]
```diff
@@ -230,7 +230,7 @@ class AuthPage extends React.Component {
 
       const gaiaBucketAddress = nextProps.identityKeypairs[0].address
       const identityAddress = nextProps.identityKeypairs[identityIndex].address
-      const gaiaUrlBase = 'https://gaia.blockstack.org/hub'
+      const gaiaUrlBase = nextProps.api.gaiaHubConfig.url_prefix
```
[the remark this answers is not in the document]
Comment (hstove) [reply app/js/auth/index.js]: Yeah, and I think I actually had something like that previously when doing the custom hub onboarding work. The truth is that if you don't configure your Gaia hub to have a trailing slash, it won't work in a few ways, and not just the browser. If your `url_prefix` is `https://myhub.com` (no slash), then you do a `putFile`, the hub server will return a `readURL` of `https://myhub.comaddress/file.json`. We almost need validation at the Gaia config level to make sure the URL prefix has a trailing slash.

[the remark this answers is not in the document]
Comment (zone117x) [reply app/js/auth/index.js]: Sounds like a larger project that should be a new issue, and we can proceed with this PR?

**Comment by hstove** [comment]
> Should we add a unit test for this situation to prevent future regressions, or is this too trivial of an issue for that?

I hate to be that guy, but unit tests for this would require a decently sized refactor that I think is better contained in #1276. It's not really possible to unit test right now, and a big refactor would, in my opinion, slow this down too much related to the benefit of getting it shipped.

**Pull request merged**